setup.py

```diff
@@ -31,12 +31,10 @@
     ],
     install_requires=[
         "invoke>=2.0.0",
-        # 1.16.2 causes a crash:
-        #   ImportError: DLL load failed while importing onnx_cpp2py_export
-        "onnx>=1.11.0,<1.16.2",
+        "onnx>=1.17.0",
```

Comment:
Word of caution: the issue with 1.16.2 was also observed by a few ppl on 1.17
https://github.com/onnx/onnx/issues/6267#issuecomment-2459924277

Reply:
Thanks for pointing that out. I made it much more permissive. We do at least have evidence that 1.17.0 works fine in our CI.